### PR TITLE
Update to Debian 10.4

### PR DESCRIPTION
**Pull request by iwanami-lin**
# Purpose of pull request
This pull request includes the following updates for update Debian from 10.3 to 10.4 .
bluez has been updated from "5.50-1" to "5.50-1.2-deb10u1"
icu has been updated from "63.1-6" to "63.1-6+deb10u1"
iputils has been updated from "20180629-2" to "20180629-2+deb10u1"
pango1.0 has been updated from "1.42.4-7-deb10u1" to "1.42.4-8-deb10u1"
ppp has been updated from "2.4.7-2+4.1" to "2.4.7-2+4.1+deb10u1"
systemd has been updated from "241-7-deb10u3" to "241-7-deb10u4"
wpa has been updated from "2.7+git20190128+0c1e29f-6+deb10u1" to "2.7+git20190128+0c1e29f-6+deb10u2"

Delete this file.
./recipes-debian/systemd/systemd/CVE-2019-15718.patch
https://security-tracker.debian.org/tracker/source-package/systemd

# Test
## How to test
$ echo 'DEBIAN_SOURCE_ENABLED="1"' >> conf/local.conf
$ bitbake core-image-minimal -g
$ bitbake bluez5
$ bitbake icu icu-native nativesdk-icu
$ bitbake iputils 
$ bitbake pango pango-native nativesdk-pango
$ echo 'DISTRO_FEATURES_append = " pam"' >> conf/local.conf
$ bitbake ppp
$ bitbake systemd
$ bitbake wpa-supplicant

## Test result
All successful.

### CODE REVIEW

**Comment by hiraku-wfs** [comment]
Thank you for the package updates.

```
$ bitbake core-image-minimal -g
```

Why did you use "-g" option here?

```
$ echo 'DISTRO_FEATURES_append = " pam"' >> conf/local.conf
$ bitbake ppp
$ bitbake systemd
$ bitbake wpa-supplicant
```

I think you appended the pam to the DISTRO_FEATURE because the ppp requires it. But it's not for systemd and wpa-supplicant, isn't it?

**Comment by hiraku-wfs** [comment]
I found the CVE-2019-15718.patch in recipes-debian/systemd/systemd/ became no longer necessary. It is resolved in this update.
https://security-tracker.debian.org/tracker/source-package/systemd

So could you also remove the file in this PR?

File: recipes-debian/sources/systemd.inc [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # This is generated by debian-source.bbclass
-DPV = "241-7~deb10u3"
+DPV = "241-7~deb10u4"
```
Comment (hiraku-wfs) [inline recipes-debian/sources/systemd.inc:2]: I found the CVE-2019-15718.patch in recipes-debian/systemd/systemd/ became no longer necessary. It is resolved in this update.
https://security-tracker.debian.org/tracker/source-package/systemd

So could you also remove the file in this PR?

**Comment by iwanami-lin** [comment]
> $ bitbake core-image-minimal -g
> Why did you use "-g" option here?

Because meta-debian-extended updates recipes-debian/ sources/*.inc.
Use -g option with DEBIAN SOURCE ENABLED="1", update recipes without building image.

> $ echo 'DISTRO_FEATURES_append = " pam"' >> conf/local.conf
> $ bitbake ppp
> $ bitbake systemd
> $ bitbake wpa-supplicant
> I think you appended the pam to the DISTRO_FEATURE because the ppp requires it. But it's not for systemd and wpa-supplicant, isn't it?

Yes, only ppp requests pam.
Should I change test method?

File: recipes-debian/sources/systemd.inc (hunk 1 above)
Comment (hiraku-wfs) [reply recipes-debian/sources/systemd.inc:2]: Confirmed. Thanks!

**Comment by hiraku-wfs** [comment]
> Because meta-debian-extended updates recipes-debian/ sources/*.inc.

Got it.

> Yes, only ppp requests pam.
> Should I change test method?

Okay. It's just confirmation. Thank you.

File: recipes-debian/sources/systemd.inc (hunk 1 above)
Comment (iwanami-lin) [reply recipes-debian/sources/systemd.inc:2]: I understand, solved this problem.

**Comment by hiraku-wfs** [comment]
Thanks!

**Pull request merged**